classes/maps.php

```diff
@@ -232,13 +227,13 @@ function initialize() {
 						var myOptions = {
 							zoom: parseInt(<?php echo esc_js( $atts['zoom_level'] ); ?>),
 							center: latlng,
-							panControl: <?php echo ( $atts['panControl'] ) ? 'true' : 'false'; ?>,
-							zoomControl: <?php echo ( $atts['zoomControl'] ) ? 'true' : 'false'; ?>,
-							scaleControl: <?php echo ( $atts['scaleControl'] ) ? 'true' : 'false'; ?>,
-							streetViewControl: <?php echo ( $atts['streetViewControl'] ) ? 'true' : 'false'; ?>,
-							mapTypeControl: <?php echo ( $atts['mapTypeControl'] ) ? 'true' : 'false'; ?>,
+							panControl: true,
+							zoomControl: true,
+							scaleControl: true,
+							streetViewControl: true,
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->